columns.html

```diff
@@ -13,9 +13,9 @@
     <title>Layouts</title>
   </head>
 
-  <body class="bg-dark">
+  <body class="bg-info">
```

Comment:
please change it to red instead of blue (as per the requirement)

Reply:
updated accordingly.